designsystem-lib/src/main/java/com/serranoie/app/designsystemlib/ui/utils/Utils.kt

```
@@ -0,0 +1,63 @@
+        }
+    }
+
+    fun dateToString(date: Date?): String {
+        if (date == null) {
+            return "Invalid date"
+        }
+        val outputFormatter = SimpleDateFormat("dd MMMM yyyy, hh:mm a", Locale.getDefault())
+        outputFormatter.timeZone = TimeZone.getDefault()
+        return try {
+            outputFormatter.format(date)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)